internal/cmd/plan.go

```diff
@@ -202,6 +202,21 @@ var planEnableOverages = &cobra.Command{
 			return err
 		}
 
+		hasPaymentMethod, err := client.Billing.HasPaymentMethod()
+		if err != nil {
+			return err
+		}
+		if !hasPaymentMethod {
+			ok, err := PaymentMethodHelperOverages(client)
+			if err != nil {
+				return fmt.Errorf("failed to check payment method: %w", err)
+			}
+			if !ok {
+				return nil
```

Comment:
You should either return an error, or print a feedback saying that overages weren't enabled.